src/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.csproj

```diff
@@ -1,7 +1,4 @@
-﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
I'd leave this dir.props in. We shoud be moving the e.g. NetCoreApp reference to use it.